examples/get_project_versions.py

```diff
@@ -66,7 +66,7 @@ def write_to_csv_file(filename, version_list):
 			if args.phase:
 				new_list = list()
 				for version in sorted_version_list:
-					if version['phase'] == phase_map[args.phase]:
```

Comment:
The loop here is on a list (sorted_version_list) and each iteration sets the value of _version_ to the next item in the list so I don't think the change proposed is correct. Maybe the error you received was due to a problem elsehwere in the code where an empty list was used in a slice or other operation that would throw the error indicated.

Reply:
This PR is just about reverting the "fix".